src/components/backend-ai-webui.ts

```diff
@@ -306,7 +306,9 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
       this.connection_server = config.general.connectionServer;
       // console.log(this.connection_server);
     }
```

Comment:
It works okay, but It seems too long to use ternary operation. How about using plain if-else statement for better readability?